sources/org.osbuild.ostree

```diff
@@ -3,7 +3,22 @@
 
 Uses ostree to pull specific commits from (remote) repositories
 at the provided `url`. Can verify the commit, if one or more
-gpg keys are provided via `gpgkeys`.
+gpg keys are provided via `gpgkeys`. The secret providers currently
+supported are:
+
+- `org.osbuild.mtls` for downloading content that requires client
+  certificats. The paths to the key and cert should be set in the
+  environment in OSBUILD_SOURCES_OSTREE_SSL_CLIENT_KEY,
+  OSBUILD_SOURCES_OSTREE_SSL_CLIENT_CERT, and optionally
+  OSBUILD_SOURCES_OSTREE_SSL_CA_CERT.
+
+- `org.osbuild.rhsm.consumer` for downloading content using RHSM
+  entitlement certificate. Secret fields `contenturl`, `consumer_cert`
+  and `consumer_key` must be set.
```

Comment:
`contenturl` isn't a secret so this line might be a bit confusing.  I'd rephrase to:
```
Secret fields `consumer_cert` and `consumer_key` must be set, as well as the `contenturl` option in the source.
```

Though, `contenturl` isn't needed for this, is it?

Reply:
Thanks, yeah `contenturl` is irrelevant my mistake.